apps/openmw/mwworld/weather.cpp

```diff
@@ -289,44 +294,54 @@ WeatherManager::WeatherManager(MWRender::RenderingManager* rendering, MWWorld::F
 
     //Weather
     Weather clear;
+    clear.mObstructsCelestialBodies = false;
```

Comment:
Regarding the memory leak: you're right. I'll reintroduce a separate object for that purpose. I wish it wasn't necessary, since the code is a lot smaller without it.

Regarding Glare View: I'll experiment tonight with different values. The name would leave me to believe it only affected the glare from the sun, but after playing with the various moon settings I would not be surprised if it did more than what says.